### DISCUSSION TITLE
Bottom toolbar: Add Volume Boost icon

### DISCUSSION THREAD

**Discussion by Trikenstein** [Goodwy/PlayBooks#97] [Ideas]
Using PlayBook Audiobook Player v4.0.3. The app has the Volume Boost, which can be enabled via the "3 dots menu" on the top right of the playing screen. This function is very useful to adjust the volume of some audiobooks having low volume.

For such a useful function, it deserves to be easily accessible in the toolbar. Currently (v4.0.3) the toolbar has a "Repeat" icon. I am not sure if the Repeat function is really useful when listening to audiobooks. I have never seen this function in other audiobook players. And personally I have never thought of a repeat function until I noticed it in your app. The most usual kind of repeat I often use is to rewind N seconds, b/c of a temporary distraction. Your app already has this function.

The idea of this post is to propose to swap the Repeat icon with the Volume Boost icon.

![Goodwy_Playbook_VolumeBoost](https://github.com/user-attachments/assets/fb98690a-c7f3-459c-a329-eaa97c445ba8)

**Comment by Goodwy** [comment]
Repeat functions is a standard feature of all players, there have been many requests for it from users. As I understand it, repeats are used for educational books.
The volume up function is a feature that is needed only for books in poor quality, usually the standard volume level is enough.

**Reply by Trikenstein** [reply]
Can you please re-open this thread. I think the idea is worth pursuing. The low volume on some books is not a "poor quality" issue. It happens from time to time, it is just the choice of a default volume set by the producer. When you listen to a variety of books produced by multiple editors, the discrepancy of the default volume level happens.

How about making the volume slider a little bit smaller and adding the Volume Boost icon at the left side?
